dev/checkstyle-suppressions.xml

```diff
@@ -50,6 +50,8 @@
               files="src/main/java/org/apache/spark/examples/JavaLogQuery.java"/>
     <suppress checks="LineLength"
               files="src/main/java/org/apache/hive/service/*"/>
+    <suppress checks="LineLength"
+              files="sql/catalyst/src/test/java/org/apache/spark/sql/catalyst/util/WkbReaderWriterTest.java"/>
```

Comment:
Please revert this because we have `// checkstyle.off: LineLength` already in that file.

Reply:
@dongjoon-hyun i tried a little bit but couldn't get it to work by 
```
// checkstyle.off: LineLength
code
// checkstyle.on: LineLength
```

I took a little look into it, I think [LineLength](https://github.com/apache/spark/blob/master/dev/checkstyle.xml#L67) is not able to be handled by the [TreeWalker](https://github.com/apache/spark/blob/master/dev/checkstyle.xml#L67) that usually handles that rule.  It's more line by line.  So i edited the checkstyle.xml to add that annotation, wdyt?

This is a problem for java files only.